.fixtures.yml

```diff
@@ -1,17 +1,17 @@
 fixtures:
   repositories:
     stdlib:
-      repo: "git://github.com/puppetlabs/puppetlabs-stdlib.git"
+      repo: "https://github.com/puppetlabs/puppetlabs-stdlib.git"
       ref:  "4.15.0"
-    ruby:    "git://github.com/puppetlabs/puppetlabs-ruby.git"
-    gcc:     "git://github.com/puppetlabs/puppetlabs-gcc.git"
-    pe_gem:  "git://github.com/puppetlabs/puppetlabs-pe_gem.git"
-    make:    "git://github.com/voxpupuli/puppet-make.git"
-    inifile: "git://github.com/puppetlabs/puppetlabs-inifile.git"
-    vcsrepo: "git://github.com/puppetlabs/puppetlabs-vcsrepo.git"
-    git:     "git://github.com/puppetlabs/puppetlabs-git.git"
+    ruby:    "https://github.com/puppetlabs/puppetlabs-ruby.git"
+    gcc:     "https://github.com/puppetlabs/puppetlabs-gcc.git"
+    pe_gem:  "https://github.com/puppetlabs/puppetlabs-pe_gem.git"
+    make:    "https://github.com/voxpupuli/puppet-make.git"
+    inifile: "https://github.com/puppetlabs/puppetlabs-inifile.git"
+    vcsrepo: "https://github.com/puppetlabs/puppetlabs-vcsrepo.git"
+    git:     "https://github.com/puppetlabs/puppetlabs-git.git"
     portage:
-      repo: "git://github.com/gentoo/puppet-portage.git"
+      repo: "https://github.com/gentoo/puppet-portage.git"
```

Comment:
oops.  I didn't actually mean to commit changes to this file.  I can remove them, but then again, they are very useful to those trying to test behind a firewall with only an http/https proxy available for Internet access.